File: src/main/java/io/zeitwert/fm/portfolio/service/api/PortfolioService.java
package io.zeitwert.fm.portfolio.service.api;

import com.google.maps.ImageResult;
import com.google.maps.model.Size;
import io.zeitwert.fm.portfolio.model.ObjPortfolio;

public interface PortfolioService {

	ImageResult getMap(ObjPortfolio portfolio, Size size);

}

File: fm-domain/src/main/java/io/zeitwert/fm/portfolio/service/api/PortfolioService.kt
package io.zeitwert.fm.portfolio.service.api

import com.google.maps.ImageResult
import com.google.maps.model.Size
import io.zeitwert.fm.portfolio.model.ObjPortfolio

interface PortfolioService {

	fun getMap(
		portfolio: ObjPortfolio,
		size: Size,
	): ImageResult?

}
